src/pint/utils.py

```diff
@@ -1580,21 +1599,206 @@ def companion_mass(pb, x, inc=60.0 * u.deg, mpsr=1.4 * u.solMass):
     return x1.to(u.Msun)
 
 
-def ELL1_check(A1, E, TRES, NTOA, outstring=True):
+@u.quantity_input
+def PBDOT(mp: u.Msun, mc: u.Msun, pb: u.d, e: u.dimensionless_unscaled):
+    """Post-Keplerian orbital decay PBDOT.
```

Comment:
Could you please add a short description - something like "compute the amount of orbital decay due to gravitational-wave losses assuming GR" - so it will be more obvious to non-expert pulsar folks what this is computing?

Reply:
sure. done.